__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const testData = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
+
+describe('GET /api/topics', () => {
+    it('200: returns an array of all the topics.', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            const {topics} = body;
+            expect(topics).toHaveLength(3);
+            topics.forEach(topic => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                });
+            });
+        });
+    });
+});
```

Comment:
lovely test for this endpoint - i would say it would be nice to include a 404 test for an invalid path but this would not have to be in the describe block as it would be a generic 404 i.e `/api/banana`